src/components/thumbnail-card/__tests__/ThumbnailCardDetails.test.js

```diff
@@ -1,13 +1,12 @@
 // @flow
 import * as React from 'react';
 
-import { mount, shallow } from 'enzyme';
+import { fireEvent, render, screen } from '@testing-library/react';
```

Comment:
we should be import these functions from here https://github.com/box/box-ui-elements/blob/master/src/test-utils/testing-library.tsx